main/pipeline/pipeline.cc

```diff
@@ -151,14 +151,20 @@ core::LocOffsets locOffset(pm_location_t *loc, pm_parser_t *parser) {
     uint32_t locStart = static_cast<uint32_t>(loc->start - parser->start);
     uint32_t locEnd = static_cast<uint32_t>(loc->end - parser->start);
 
-    std::cout << "locStart: " << locStart << std::endl;
-    std::cout << "locEnd: " << locEnd << std::endl;
-
     return core::LocOffsets{locStart, locEnd};
 }
 
 unique_ptr<parser::Node> convertPrismToSorbet(pm_node_t *node, pm_parser_t *parser, core::GlobalState &gs) {
     switch (PM_NODE_TYPE(node)) {
+        case PM_ELSE_NODE: {
+            auto elseNode = reinterpret_cast<pm_else_node *>(node);
+
+            if (elseNode->statements != nullptr) {
+                return convertPrismToSorbet(reinterpret_cast<pm_node *>(elseNode->statements), parser, gs);
+            }
+
+            return nullptr;
```

Comment:
I think using the early return for the null case makes this a bit clearer:

```suggestion
            if (elseNode->statements == nullptr) return nullptr;

            return convertPrismToSorbet(reinterpret_cast<pm_node *>(elseNode->statements), parser, gs);
```